app/controllers/application_chatops_controller.rb

```diff
@@ -8,4 +8,13 @@ def verify_slack_hmac
   rescue Slack::Events::Request::MissingSigningSecret, Slack::Events::Request::InvalidSignature, Slack::Events::Request::TimestampExpired
     render plain: "Nope.", status: :unauthorized
   end
+
+  def subcommand
+    Utils::ParsesCliStyleCommandArgs.new.call(text: params[:text]).subcommand
+  end
+
+  def command_params
```

Comment:
The intent here is for arguments passed to a slash command to feel like typical rails `params` hash.